opa-manager-operator/docs/gatekeeper-rb.yaml

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
This is just a pre-rendered gatekeeper-rb.yaml.template, remove it?

Reply:
Please remove this.

Reply:
removed